swift/ql/lib/codeql/swift/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -296,6 +300,22 @@ import Cached
 /** Holds if `n` should be hidden from path explanations. */
 predicate nodeIsHidden(Node n) { n instanceof FlowSummaryNode }
 
+private class DictionarySubscriptNode extends NodeImpl, TDictionarySubscriptNode {
```

Comment:
I think a QLDoc on this class would help future developers.

Reply:
cc @rdmarsh2

Reply:
This QLDoc has been added now and LGTM.